packages/g/gimp-help/package.yml

```diff
@@ -1,72 +1,163 @@
 name       : gimp-help
```

Comment:
The commit title and PR title needs to have lowercase `gimp-help`, not `Gimp-help`.

Reply:
Done, I think it's fixed now

Reply:
?